<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>khamsobo</class>
 <widget class="QMainWindow" name="khamsobo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>-10</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Thong tin benh nhan</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Do tuoi</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tuoi"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ten"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ten</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>461</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Dien thong tin kham</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>30</y>
       <width>64</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chỉ số tim:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="cs_tim">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>30</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>58</y>
       <width>99</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chỉ số huyết áp:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="csha">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>58</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>86</y>
       <width>38</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chú ý:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="chuy">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>86</y>
       <width>199</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QListView" name="listView_benhduocchon">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>117</y>
       <width>199</width>
       <height>137</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QTreeView" name="treeView_benh">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>117</y>
       <width>220</width>
       <height>137</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>500</height>
      </size>
     </property>
    </widget>
    <widget class="QTreeView" name="treeView_benhdk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>220</width>
       <height>138</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>220</height>
      </size>
     </property>
    </widget>
    <widget class="QTreeView" name="treeView_thuocgoiY">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>199</width>
       <height>138</height>
      </rect>
     </property>
    </widget>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QPushButton" name="khongduak">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>173</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Chưa đủ sức khỏe để tiêm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>183</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Danh sách bệnh nhân tiếp theo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>570</y>
      <width>159</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Đã đủ sức khỏe để tiêm</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_dsCho">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>570</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>tach chu</string>
    </property>
   </widget>
   <widget class="QListView" name="danhsach_chonthuoc">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
